util/incoming-connections.js

```diff
@@ -25,11 +25,10 @@ module.exports = function (config) {
     return defaultPort
   }
 
-
-  //legacy configuration didn't have a scopes concept,
-  //so interpret that as every scope at once.
-  //I think there is probably a better way to do this,
-  //but am fairly sure this will probably work.
+  // legacy configuration didn't have a scopes concept,
+  // so interpret that as every scope at once.
+  // I think there is probably a better way to do this,
+  // but am fairly sure this will probably work.
```

Comment:
Code style changes are best done in their own commit, it makes it significantly easier to review (I can browse the commits and focus my attention only on the non-style commits), and makes the git blame process easier too.